Makefile

```diff
@@ -50,3 +50,11 @@ clear-cache:
 	docker rm -vf $$(docker ps -a -q) 2>/dev/null ; \
 	docker rmi -f $$(docker images -a -q) 2>/dev/null ; \
 	docker volume rm $$(docker volume ls -q) 2>/dev/null ; true
+
+experiment=*
+
+stop-trials:
+	docker rm -vf $$(docker ps -f "name=fuzzbench-r-$(experiment)-[0-9]+$$" -q) 2>/dev/null ; true
```

Comment:
why is the `; true` part necessary? why would the command fail, and why would we not terminate / hide the error code?

if it is necessary, the proper way to do this in a Makefile is to [prefix the command](https://www.gnu.org/software/make/manual/html_node/Errors.html) with "-", ie.:
```
  - docker rm ...
```

Reply:
When there is no containers to remove, there is some error showing up.
Adding those will help to get a clean console I guess.
```
docker rm --volumes --force $(docker ps --filter "name=r-*-[0-9]+$" -q)
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
make: *** [Makefile:57: stop-trials] Error 1
```
If we add `-` in front of it, the error report is the same but adding one ignore at last:
```
docker rm --volumes --force $(docker ps --filter "name=r-*-[0-9]+$" -q)
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
make: [Makefile:58: stop-trials] Error 1 (ignored)
```
If we add only `2>/dev/null` then
```
docker rm --volumes --force $(docker ps --filter "name=r-*-[0-9]+$" -q) 2>/dev/null
make: *** [Makefile:57: stop-trials] Error 1
```
Thus, we add `true` at last to get rid of all error reporting.

Reply:
Thanks! I don't see why hiding error messages and masking the status code is useful in this case: if I run `make kill-trials`, and no trials were killed, I'd like to know it.

Reply:
Looks like a valid need for potential users. I thought no error reporting is more smooth in general. 

Shall we also change it for letting users know there are no trials were killed for `make clear-cache` too?
https://github.com/google/fuzzbench/blob/7d8527cbded8d0280cbed461058a7b89beb935cd/Makefile#L48-L52